src/zcertstore.c

```diff
@@ -267,6 +267,17 @@ zcertstore_print (zcertstore_t *self)
     }
 }
 
+//  --------------------------------------------------------------------------
+//  Return the state stored in certstore
+void *
+zcertstore_get_state (zcertstore_t *self)
+{
+    if (self)
+        return self->state;
+
+    return NULL;
```

Comment:
Our code style advocates to fail fast.
```c
assert (self);
return self->state;
```

Reply:
It would be great if you could change this in another PR

Reply:
Done

Reply:
I saw your update. Thanks for fixing it. I will keep that in mind for future pull requests.